### PR TITLE
Add requirements.txt to list python script dependency.

### PR DESCRIPTION
**Pull request by jiejanezhang**
Add requirements.txt to list python script dependency.

### CODE REVIEW

**Review by intwanghao — APPROVED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+graphviz==0.20.3
```
Comment (zhimingwang36) [inline requirements.txt:1]: Should this file be added into install folder?  (If so, need to update the install target to include it).

Comment (zhimingwang36) [inline requirements.txt:1]: can other version of graphviz also works?

Comment (jiejanezhang) [reply requirements.txt:1]: Feedback from Sergey is that it is better to have explicit version and maintain it up-to-date manually.

Comment (jiejanezhang) [reply requirements.txt:1]: Yes.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**